community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -191,9 +224,8 @@
 
     @Description( "The maximum amount of time to wait for running transactions to complete before allowing "
                   + "initiated database shutdown to continue" )
-    @Internal
     public static final Setting<Long> shutdown_transaction_end_timeout =
-            setting( "unsupported.dbms.shutdown_transaction_end_timeout", DURATION, "10s" );
+            setting( "dbms.shutdown_transaction_end_timeout", DURATION, "10s" );
```

Comment:
Don't we need migration for this? Feels like we should make things easy for (albeit few) existing users of an unsupported setting when we promote such to a supported setting.